### DISCUSSION TITLE
Change default screenshot format to jpeg

### DISCUSSION THREAD

**Discussion by kamil1014** [allure-framework/allure2#2040] [Questions & Support] [answered]
Hi!
I have a problem with size of my report, is there a way to change format of default screenshots that are made by Allure? We can set format for those that are added by addAtachament but is there a way to change that default one?

**Comment by baev** [comment, marked as answer]
Allure does not convert attached screenshots. Allure only displays attached data (all attachments are bytes for Allure) in proper views. You can control the size and format for all the screenshots by yourself, passing desired bytes to the `addAtachament` method.
